include/StandardDeviationHeuristics.h

```diff
@@ -0,0 +1,21 @@
+#ifndef STANDARDDEVIATIONHEURISTICS_H
+#define STANDARDDEVIATIONHEURISTICS_H
+#include "MeshAmalgamationBase.h"
+class StandardDeviationHeuristics : public MeshAmalgamation {
+private:
+  double _standard_deviation_tolerance =0.05;
+
+protected:
+public:
+  StandardDeviationHeuristics(libMesh::Mesh& mesh,
```

Comment:
This looks like it's actually a threshold heuristic that you happen to apply to a std dev in this case.  Perhaps implement it more generically.